### PR TITLE
[FAB-17] Add touch support

### PR DESCRIPTION
**Pull request by nicolauscg**
replace mouse events with pointer events which handles both touch and mouse, https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events

### CODE REVIEW

**Review by daystram — APPROVED** [review]

**Comment by nicolauscg** [comment]
Danny tested on ios mobile browser, if canvas is pressed for a while without moving for a bit (like 2s), behavior is similar to "select all" which highlights all text

I tested on android mobile and its fine, on browser emulating mobile touch, doing scenario above leads to opening a context menu with options: save image, copy image and inspect.

**Comment by nicolauscg** [comment]
addressed multi touch and hold opening context menu on safari issue, awaiting rereview

File: fableous-fe/src/components/canvas/Canvas.tsx [hunk 1]
```diff
@@ -81,8 +86,14 @@ const Canvas = forwardRef<HTMLCanvasElement, CanvasProps>(
         ctx.lineJoin = "round";
         ctx.strokeStyle = targetColor;
         ctx.lineWidth = targetWidth;
-        ctx.moveTo(x1, y1);
-        ctx.lineTo(x2, y2);
+        if (x1 !== x2 || y1 !== y2) {
```
Comment (daystram) [inline fableous-fe/src/components/canvas/Canvas.tsx:89]: Replace with ternary operator?

**Review by daystram — APPROVED** [review]

File: fableous-fe/src/components/canvas/Canvas.tsx (hunk 1 above)
Comment (nicolauscg) [reply fableous-fe/src/components/canvas/Canvas.tsx]: done

**Review by daystram — APPROVED** [review]

**Pull request merged**